README.md

```diff
@@ -1,2 +1,179 @@
-# Leaderboard
-This is a Leaderboard project
+<a name="readme-top"></a>
+
+<div align="center">
+
+<!-- MAIN HEADING -->
+
+  <h3><b>LeaderBoard Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Portfolio <a name="about-project"></a>
+
+> This is a Leaderboard project with just HTML, Javascript and CSS using ES6
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. Linters
+3. JavaScript ES6
+4. Webpack
+### Tech Stack <a name="tech-stack"></a>
+
+> This is a version with mono view Desktop 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">Linters</a></li>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Added Linters
+> - Added ES6
+> - Added HTML
+> - Added Javascript
+> - Added HTML syntx from JS
+> - Webpack 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo" target="_blank"></a>
+
+- [Click here to see it Live ](https://ishimwepati.github.io/Leaderboard/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Follow the step bellow to get the project localy.
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- SETUP -->
+### Setup
+
+Clone this repository to your desired folder using:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ishimwepati/Leaderboard.git
+```
+<!-- INSTALL -->
+
+### Install
+
+Make sure you install this project with the latest version of Git and a code editor (VS is goog).
+
+### Usage
+
+To run the project, execute the following command:
+
+Browser up your project folder location, once there type the following command using your terminal ### code . all the project files will open and VS.
+
+### Deployment
+
+You can deploy this project using:
+>1.  A Code Editor
+>2. Latest version of Git
+
```

Comment:
- [ ] Tpo make your documentaation looks more professional and descriptive, I think it would be better if you update some of the sub-sections of the `Getting Started` main section with appropriate and correct content information for each one, such as how to `install` your app or dependencies, `run build` for production, `run` your app,... To help you, I suggest you refer **[this gist_file](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)** as it includes most of information you need.

Reply:
Thanks a lot for this.